src/marten/db/field/slug.cr

```diff
@@ -3,8 +3,10 @@ require "./string"
 module Marten
   module DB
     module Field
-      # Represents an slug field.
+      # Represents a slug field.
       class Slug < String
+        private getter slugify
```

Comment:
Can we move private getter definitions closer to other private methods/constants? 🙏